docs/validators/nodes-scripts/install-update.md

```diff
@@ -5,16 +5,20 @@ title: Installing a Validator Node
 
 ## **Install your node(s)**
 
-After preparing the user permissions, the script configurations, and the keys, the actual node installation can begin. The Validator script is a multi-purpose tool for managing your node, it is accessible to Devnet and Testnet Network.
-
-- _Note: This process installs a validator node for all MultiversX networks (mainnet, devnet, and testnet)._
+After preparing the user permissions, the script configurations, and the keys, the actual node installation can begin. The Validator script is a multi-purpose tool for managing your node, it is accessible to Mainnet, Devnet or Testnet Network.
```

Comment:
can you remove `Network` word from the end of the sentence?

Reply:
done